prepublish.sh

```diff
@@ -6,13 +6,18 @@ set -e
 
 REPO=js-builder
 
-# Make sure that the local branch tip revision is the same 
-# as upstream master.
-LOCAL_TIP=$(git rev-parse HEAD)
-REMOTE_TIP=$(git ls-remote https://github.com/jenkinsci/$REPO master | awk '{ print $1 }')
-
-if [ $LOCAL_TIP != $REMOTE_TIP ]
+if [ "$npm_config_tag" != "latest" ]
```

Comment:
`$npm_config_tag` needs to be in the `env`? If so, maybe hint at that in the message at line 20?